pkg/systemd/generate/pods.go

```diff
@@ -256,14 +256,14 @@ func generatePodInfo(pod *libpod.Pod, options entities.GenerateSystemdOptions) (
 	return &info, nil
 }
 
-// Unless already specified, the pod's exit policy to "stop".
-func setPodExitPolicy(cmd []string) []string {
+// Determine whether the command array includes an exit-policy setting
+func hasPodExitPolicy(cmd []string) bool {
```

Comment:
Is this rewrite really needed? Could have moved it caller just one line above.

Reply:
What do you mean by moving the caller a line above?